assets/large_language_models/components_pipelines/oai_v2_1p/openai_completions_finetune_pipeline/spec.yaml

```diff
@@ -63,13 +73,15 @@ jobs:
       model: ${{parent.inputs.model}}
   openai_completions_finetune:
     type: command
-    component: azureml://registries/azure-openai-v2/components/openai_completions_finetune/versions/0.3.5
+    component: azureml://registries/azure-openai-v2/components/openai_completions_finetune/versions/0.3.10
     inputs:
       input_dataset: ${{parent.jobs.openai_data_import.outputs.out_dataset}}
       model: ${{parent.inputs.model}}
       task_type: ${{parent.inputs.task_type}}
       registered_model_name: ${{parent.inputs.registered_model_name}}
-      epochs: ${{parent.inputs.epochs}}
+      n_epochs: ${{parent.inputs.n_epochs}}
```

Comment:
Is this not a breaking change?

Reply:
Yes. We will follow up with the corresponding team for making the appropriate changes that are needed as a result of this. Currently there are no customers for this pipeline component so there is no impact.